salt/journal/config/srv-journal-app-config-parameters.yml

```diff
@@ -1,5 +1,16 @@
+{% set on_elasticache = salt['elife.cfg']('cfn.outputs.ElastiCacheHost') %}
+{% set standard_keys = ['secret', 'api_key', 'api_url', 'api_url_public', 'side_by_side_view_url', 'session_name', 'status_checks', 'gtm_id', 'disqus_domain'] %}
+
 parameters:
-    {% for name in ['secret', 'api_key', 'api_url', 'api_url_public', 'side_by_side_view_url', 'session_name', 'status_checks', 'gtm_id', 'disqus_domain', 'redis_cache', 'redis_http_cache'] %}
+    {% if not on_elasticache %}
+    {% do standard_keys.extend(['redis_cache', 'redis_http_cache']) %}
+    {% else %}
+    {% set remote_redis = "redis://" + salt['elife.cfg']('cfn.outputs.ElastiCacheHost') + ":" + salt['elife.cfg']('cfn.outputs.ElastiCachePort') %}
+    redis_cache: {{ remote_redis }}
+    redis_http_cache: {{ remote_redis }}
```

Comment:
I had envisaged having separate Redises (Redi?), hence splitting out `redis_http_cache`. Need to check what happens on a deploy (ultimately we want sessions here, which we really don't want cleared on a deploy etc). If they are treated separately enough through prefixes we can revert to having just the one parameter.